src/pages/Gallery/Gallery.module.scss

```diff
@@ -0,0 +1,264 @@
+    width: 14rem;
+    height: 14rem;
+  }
+}
+
+.galleryListItem {
+  background-color: rgba(255, 255, 255, 1);
+  border-color: rgba(0, 0, 0, 1);
+  border-width: 2px;
+  padding-top: 0.5rem;
+  text-align: center;
+}
+
+.imageTitle {
+  color: rgba(0, 0, 0, 1);
+}
+
+.galleryListItemContainer {
+  margin-top: 1.5rem;
+  margin-bottom: 0rem;
+
+  @media (min-width: 1280px) {
+    margin-top: 2.5rem;
+    margin-bottom: 0rem;
+  }
+}
+
+.counterContainer {
+  display: inline-block;
+  position: relative;
+  text-align: left;
+}
+
+.legendContainer {
+  margin-top: 0.5rem;
+  margin-bottom: 0rem;
+  align-items: center;
+}
+
+/// @todo Remove this class after icon decision.
+.delete1 {
+  opacity: 1;
+  transform: translateX(0) translateY(0) rotate(0) skewX(0) skewY(0) scaleX(1) scaleY(1);
+}
+
+/// @todo Remove this class after icon decision.
+.delete2 {
+  opacity: 0;
+  transform: translateX(0) translateY(0) rotate(0) skewX(0) skewY(0) scaleX(1) scaleY(1);
+}
+
+.dropdownContentContainer {
+  background-color: rgba(255, 255, 255, 1);
+  border-color: rgba(0, 0, 0, 1);
+  border-right-width: 2px;
+  border-bottom-width: 2px;
+  border-left-width: 2px;
+  position: absolute;
+  left: 0rem;
+  width: 169px;
+  z-index: 10;
+  transform-origin: top;
+}
+
+.dropdownOpen {
+  transition-property: background-color, border-color, color, fill, stroke, opacity, box-shadow,
+    transform;
+  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+  transition-duration: 150ms;
+  opacity: 1;
+}
+
+.dropdownClose {
+  transition-property: background-color, border-color, color, fill, stroke, opacity, box-shadow,
+    transform;
+  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+  transition-duration: 150ms;
+  opacity: 0;
+}
+
+.transform {
+  transform: translateX(0) translateY(0) rotate(0) skewX(0) skewY(0) scaleX(1) scaleY(1);
+}
+
+.dropdownButton {
+  background-color: rgba(255, 255, 255, 1);
+  border-collapse: collapse;
+  border-color: rgba(0, 0, 0, 1);
+  display: inline-flex;
+  justify-content: center;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 1.25rem;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  padding-left: 0.25rem;
+  color: rgba(55, 65, 81, 1);
+  text-transform: uppercase;
+  width: 169px;
+  border-width: 2px;
+
+  &:not(.countLabel):hover &:hover {
+    background-color: rgba(249, 250, 251, 1);
+  }
+}
+
+.countLabel {
+  width: 143px;
+}
+
+.linesContainer {
+  border-color: rgba(0, 0, 0, 1);
+  border-bottom-width: 2px;
+  padding-bottom: 0.5rem;
+}
+
+.pageTitle {
+  font-weight: 800;
+  font-size: 1.875rem;
+  line-height: 2.25rem;
+  text-align: center;
+  color: rgba(0, 0, 0, 1);
+  text-transform: uppercase;
+  letter-spacing: -0.025em;
+
+  @media (min-width: 640px) {
+    font-size: 2.25rem;
+    line-height: 2.5rem;
+  }
+}
+
+.dropdownContent {
+  padding-top: 0.25rem;
+  padding-bottom: 0.25rem;
+}
+
+.galleryContainer {
+  margin-top: 3rem;
+  margin-bottom: 0rem;
+}
+
+.viewButton {
```

Comment:
the button should be visible on card hover, I think it should work like:
click on card -> go to detail
click on button -> go to opensea directly

Would be also good to use <Button> component